SyntaxError: invalid string prefix in ldap3/utils/port_validators.py"

When running tests in Travis CI with Python 3.9a5, I get:
```
Downloading archive: https://storage.googleapis.com/travis-ci-language-archives/python/binaries/ubuntu/16.04/x86_64/python-3.9-dev.tar.bz2
$ curl -sSf --retry 5 -o python-3.9-dev.tar.bz2 ${archive_url}
$ sudo tar xjf python-3.9-dev.tar.bz2 --directory /
$ git clone --depth=50 --branch=master https://github.com/univention/kelvin-rest-api-client.git univention/kelvin-rest-api-client
$ source ~/virtualenv/python3.9-dev/bin/activate
$ python --version
Python 3.9.0a5+

[..]

tests/conftest.py:26: in <module>
    from ldap3 import (
.tox/py39/lib/python3.9/site-packages/ldap3/__init__.py:138: in <module>
    from .core.server import Server
.tox/py39/lib/python3.9/site-packages/ldap3/core/server.py:38: in <module>
    from ..utils.port_validators import check_port, check_port_and_port_list
E     File "/home/travis/build/univention/kelvin-rest-api-client/.tox/py39/lib/python3.9/site-packages/ldap3/utils/port_validators.py", line 20
E       return'Cannot specify both a source port and a source port list'
E            ^
E   SyntaxError: invalid string prefix
```

The missing space between `return` and `'Cannot...` makes Python think all chars of `return` are mean as string prefixes, but `etun` are unknown as that.
Interestingly I cannot reproduce this outside of Travis CI.
Although it seems to be usually ignored, it _is_ a syntax error.

--- a/ldap3/utils/port_validators.py
+++ b/ldap3/utils/port_validators.py
@@ -17,7 +17,7 @@ def check_port_and_port_list(port, port_list):
     Return an error message indicating what is invalid if something isn't valid.
     """
     if port is not None and port_list is not None:
-        return'Cannot specify both a source port and a source port list'
+        return 'Cannot specify both a source port and a source port list'
     elif port is not None:
         if isinstance(port, int):
             if port not in range(0, 65535):

